### DISCUSSION TITLE
Moving Dockerhub repo - feedback wanted

### DISCUSSION THREAD

**Discussion by snipe** [snipe/snipe-it#10072] [Q&A]
Peeps who use docker IRL, if the dockerhub repo location were to change, how much of a hassle is it to you? Future plans are to move snipe/snipe-it to grokability/snipe-it & we want to get a feel for how much lead time would be needed & how much it might monkey folks up.

**Comment by ghost** [comment]
For my Organization no problem, we only need to know it.

Anyway thanks for you work, Snipe-It is a great Software :)

Just a little Thing: there will be the tag 5.2.0 available in the actual repo?

We prefer to not use the latest tag, so we can better manage upgrades.

(i know, is off topic)

**Comment by sebhz** [comment]
Exact same feedback from here - pulling the docker image from another dockerhub repo is not a problem.
We do not really need to know in advance as well: just tell when the move has occurred and we'll start pulling updates from the new place.

And same off-topic feedback: could you please push the 5.2.0 release docker image with the appropriate tag in the current repo ?
We really prefer to align on "official" release tags rather than latest or (God forbids !) develop.

**Comment by fabricat** [comment]
Same as above: it is not a big deal. Just put the change into evidence in the release notes 👍
